project2/ViewController.swift

```diff
@@ -131,5 +133,59 @@ class ViewController: UIViewController {
         guard let newHigh = highScoreArray.max() else { return }
         highScore = newHigh
     }
+    
+    func registerLocal () {
```

Comment:
If this function is not intended to be called outside of this scope, then maybe we should set it private(includes all other functions as well)